packages/core/database/src/utils/identifiers/index.ts

```diff
@@ -30,19 +35,18 @@ export type NameOptions = {
 };
 
 // Fixed compression map for suffixes and prefixes
-// TODO: fix this in the migration
 const replacementMap = {
-  // links: 'lnk',
-  // order_inv_fk: 'oifk',
-  // order: 'ord',
-  // morphs: 'mph',
-  // index: 'idx',
-  // inv_fk: 'ifk',
-  // order_fk: 'ofk',
-  // id_column_index: 'idix',
-  // order_index: 'oidx',
-  // unique: 'uq',
-  // primary: 'pk',
+  links: 'lnk',
+  order_inv_fk: 'oifk',
+  order: 'ord',
+  morphs: 'mph',
+  index: 'idx',
+  inv_fk: 'ifk',
+  order_fk: 'ofk',
+  id_column_index: 'idix',
```

Comment:
shouldn't this be shortened to something like this, i mean looking at how we are shortening index everywhere else
```suggestion
  id_column_index: 'ididx',
```

Reply:
I used `idix` because that's what was in the RFC but I have no idea where that came from or why. If anyone wants to vote here, I'll go with whatever the consensus is, I don't think it's super important as long as we document it.